bloom-instance/service/backend/sg.tf

```diff
@@ -0,0 +1,28 @@
+
+# Enable outbound connections from service to DB port
+resource "aws_vpc_security_group_egress_rule" "from_alb" {
+  security_group_id = module.service.security_group.id
+
+  description = "Allow service ${local.name} to access database"
+  cidr_ipv4   = "0.0.0.0/0"
+  from_port   = var.db.port
+  to_port     = var.db.port
+  ip_protocol = "tcp"
+
+  tags = var.additional_tags
+}
+
+# Enable inbound connections to DB from service
+resource "aws_vpc_security_group_ingress_rule" "from_alb" {
+  security_group_id = var.db.security_group_id
+
+  description = "Allow service ${local.name} to access database"
+  from_port   = var.db.port
+  to_port     = var.db.port
+  ip_protocol = "tcp"
+  #referenced_security_group_id = module.service.security_group.id
```

Comment:
is this supposed to be uncommented?

Reply:
Possibly in the future.  Originally it was used to restrict access to just the service's security group, but I opened it up to all internal IPs because locking it to the service it made it impossible to verify database state from my bastion host.  We'll want to uncomment this in the future (and remove the line below) once we are able to add in other rules for SGs that can be assumed by bastion hosts, etc.  Possibly an extra bit of config passed into the db module